### DISCUSSION TITLE
Use TAP?

### DISCUSSION THREAD

**Discussion by aragilar** [pmodwrc/fits2db#4] [General]
https://www.ivoa.net/documents/TAP/

**Comment by gosow9** [comment]
Hi there 
If I understood TAP right it is more a tool to query different tables that are already in one or multiple db's.

This tool is more an approach to load and manage the data into a db from fits files in an db agnostic way. Once in the db 

If I missed something and there is a cool way to extract data from fits files and load them into an db and update if changes are made in the original file I'm really interested :)
